README.md

```diff
@@ -71,5 +87,3 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job @NatanimA 🤝

- [ ] [Optional] Kindly fix the MIT license since the link is broken. This will make your readme file more professional.